cmd/pull.go

```diff
@@ -43,14 +45,18 @@ var pullCmd = &cobra.Command{
 		//build the module and name query paramaters
 		query := url.Values{}
 
+		if noModule && module == "" {
```

Comment:
Don't we want to get rid of the module check here too? You should be able to provide the noModule flag as well as provide other modules as well.

Reply:
@plusplusben I don't think I was thinking in terms of a set when I made this change, so yeah you're right here.